packages/portal/docs/style/BootstrapVue/Tooltip.md

```diff
@@ -0,0 +1,36 @@
+Customised BootstrapVue tooltip component.
+
+Docs including more features: https://bootstrap-vue.org/docs/components/tooltip
+
+Used by:
```

Comment:
Order these alphabetically, and link to them where also present in the style guide.